src/main/java/gregtech/api/capability/impl/SteamMultiWorkable.java

```diff
@@ -23,15 +23,26 @@ public ParallelLogicType getParallelLogicType() {
     }
 
     @Override
-    public boolean shouldTrimOutputs() {
+    public boolean canVoidRecipeOutputs() {
         return true;
     }
 
+    @Override
+    public boolean canDoChancedOutputs() {
+        return false;
+    }
```

Comment:
This is not really what I wanted for the steam multis. I still want the steam multis to have the ability to have chanced outputs, but only in the case where there were no existing non-chanced outputs.

My goal behind introducing output trimming is:
- If there is 1 recipe output, and no chanced outputs, do nothing
- If there are > 1 recipe outputs (non-chanced), trim the recipe outputs to 1 and wipe any chanced outputs
- If there are no recipe outputs and just chanced outputs, trim the chanced outputs to 1

With this `canDoChancedOutputs` method, I don't think the above can be met, as if the recipe just has chanced outputs, it would refuse to run, instead of trimming the chanced outputs to 1.

In addition, I personally don't really think we need the `canDoChancedOutputs`, as I believe that the chanced outputs should always be considered for a recipe, unless the multi desires to trim/clamp the outputs, which we already have a method for.